e2e/playwright/test-utils.ts

```diff
@@ -150,6 +158,19 @@ export async function closePane(page: Page, testId: string) {
 
 async function openKclCodePanel(page: Page) {
   await openPane(page, 'code-pane-button')
+
+  // Code Mirror lazy loads text! Wowza! Let's force-load the text for tests.
```

Comment:
I think it uses a transaction-based system that debounces. This is a smart workaround until we can properly await that loaded state!

Reply:
I'm not sure, it really, really looked like lazy loading behavior to me but I could be wrong